src/pharmpy/model/external/nonmem/table.py

```diff
@@ -73,10 +73,10 @@ def _parse_table(
             table = NONMEMTable(''.join(content))  # Fallback to non-specific table type
 
         if table_title_line is not None:
-            m = re.match(r'TABLE NO.\s+(\d+)', table_title_line)
-            if not m:
+            table_number_match = re.match(r'TABLE NO.\s+(\d+)', table_title_line)
```

Comment:
The name `m` for a match is standard in Python so this change makes things less clear in my opinion.